src/plugins_3rdparty/kalign/src/KalignTask.cpp

```diff
@@ -112,6 +111,11 @@ Task::ReportResult KalignTask::report() {
     return ReportResult_Finished;
 }
 
+bool KalignTask::isAlphabetSupported(const QString &alphabetId) {
+    return !(alphabetId == BaseDNAAlphabetIds::RAW() ||
```

Comment:
Could you please change the alphabet check here: check for alphabets KAlign support instead of those KAlign does not support?
If we add a new alphabet tomorrow your check may not be safe.